### ISSUE TITLE
Readonly apidb chunking

### ISSUE THREAD

**Issue by pnorman** [zerebubuth/openstreetmap-cgimap#71]
Currently for requesting nodes (and ways/relations) the readonly backend requests nodes with essentially 

``` sql
SELECT *
  FROM current_nodes
  WHERE id IN (... nodes ...);
```

The optimizer is smart enough to turn `id IN (... nodes ...)` to `id = ANY({... nodes ...})`.

It takes the lists of nodes and splits them into chunks of 5k objects and requests each chunk sequentially.

I tested with pgsnapshot where this query is virtually identical with a set of 14k nodes, then with the set split into two 7k sets.

For 14k the time was 40.35ms to run the query (std dev 0.18ms) while for 7k+7k the total time to run the query was 41.95ms (std dev 0.10ms). This doesn't include query planning times or communication times, but those will favour running one query.

Are we gaining any benefits from splitting into chunks?

We begin to serialize _marginally_ faster on _large_ map calls, decreasing the time to the first XML by (large call time - single small call time), at the cost of additional code complexity, additional DB load, and slower overall response.

I would like to remove the chunking. Thoughts?

**Comment by zerebubuth** [comment]
This is probably all related in one big tangle with not only #88 as you've already pointed out but #123 as well.

**Comment by zerebubuth** [comment]
@pnorman since #78 was merged, and given the existence of #88 and #123 - do you consider this issue closed? Is there something we should break out as a new issue?

**Comment by pnorman** [comment]
I find no reference to 5000, so looks like all the chunking is gone.
